docker-machine-nfs.sh

```diff
@@ -146,15 +146,15 @@ configureNFS()
     echoSuccess "\t\t\t\t\t\tOK"
 }
 
-# @info:    Configures the Boot2Docker to mount nfs
-configureBoot2Docker()
+# @info:    Configures the Docker Machine to mount nfs
+configureDockerMachine()
```

Comment:
Well docker-machine is the layer above it which encapsulates all the internals, whether it be virtualbox, aws, azure, ... 

Currently the "virtualbox" mode still uses boot2docker, and it's what gets configured.

When you do `docker-machine ssh box-name` you see boot2docker in big.

For this, the doc is still accurate.